### ISSUE TITLE
Loss not move

### ISSUE THREAD

**Issue by jinyu121** [jinyu121/Tensorflow-ShuffleNet#1]
```
INFO:tensorflow:global step 10: loss = 102.3811 (0.078 sec/step)
INFO:tensorflow:global step 20: loss = 133.8150 (0.080 sec/step)
INFO:tensorflow:global step 30: loss = 5.7762 (0.077 sec/step)
INFO:tensorflow:global step 40: loss = 5.2986 (0.080 sec/step)
INFO:tensorflow:global step 50: loss = 5.2983 (0.076 sec/step)
INFO:tensorflow:global step 60: loss = 5.2983 (0.077 sec/step)
INFO:tensorflow:global step 70: loss = 5.2983 (0.076 sec/step)
INFO:tensorflow:global step 80: loss = 5.2983 (0.080 sec/step)
INFO:tensorflow:global step 90: loss = 5.2983 (0.089 sec/step)
INFO:tensorflow:global step 100: loss = 5.2983 (0.083 sec/step)
INFO:tensorflow:global step 110: loss = 5.2983 (0.075 sec/step)
INFO:tensorflow:global step 120: loss = 5.2983 (0.077 sec/step)
INFO:tensorflow:global step 130: loss = 5.2983 (0.076 sec/step)
INFO:tensorflow:global step 140: loss = 5.2983 (0.076 sec/step)
INFO:tensorflow:global step 150: loss = 5.2983 (0.076 sec/step)
```
